solid/src/main/java/com/inrupt/client/solid/SolidClient.java

```diff
@@ -284,8 +299,19 @@ public <T extends Resource> CompletionStage<Void> delete(final T resource, final
             if (isSuccess(res.statusCode())) {
                 return null;
             } else {
-                throw SolidClientException.handle("Unable to delete resource", resource.getIdentifier(),
-                        res.statusCode(), res.headers(), new String(res.body(), UTF_8));
```

Comment:
Note the UTF_8 change here. Is this ok?

Reply:
Definitely want UTF_8. Otherwise it defaults to the system defined encoding

Reply:
Good catch, fixed with 43f3a02196